src/birthder.js

```diff
@@ -209,12 +243,35 @@
     return message
   }
 
+  /**
+   * Check if there is a birthday yesterday
+   * @param {Robot} robot Hubot instance
+   * @returns {Void}
+   */
+  async function checkYesterday (robot) {
```

Comment:
I guess the name `removeExpiredBirthdayChannels` would reflect the purpose of the function better.